### DISCUSSION TITLE
打开后没有导入出装设置

### DISCUSSION THREAD

**Discussion by Sea-of-star** [cangzhang/ChampR#293] [General]
只显示这些，并且界面一闪一闪，代理和PowerShell都查看过，都没有问题
![image](https://github.com/cangzhang/champ-r/assets/63353503/8c0fe83d-a03c-4227-9cb4-98fa44ea2600)

**Comment by cangzhang** [comment]
如果是台服可以看这里 https://github.com/cangzhang/champ-r/discussions/292#discussioncomment-6808083
